src/components/AuthForm.js

```diff
@@ -194,7 +215,7 @@ export default class AuthForm extends Component {
     };
   }
 
-  onChange = ({formData}) => {
+  onChange = ({formData}: {formData: Object}) => {
```

Comment:
It would be nice to manage using the `AuthData` union type here, but couldn't have it working :/